code/modules/mob/living/carbon/human/species/station/traits/neutral.dm

```diff
@@ -69,6 +69,23 @@
 	..(S,H)
 	H.verbs |= /mob/living/carbon/human/proc/bloodsuck
 
+/datum/trait/vampire
+	name = "Vetalan / Vampiric"
+	desc = "Vampires, officially known as the Vetalan, are weaker to burns, bright lights, and must consume blood to survive. To this end, they can see near-perfectly in the darkness, possess sharp, numbing fangs, and anti-septic saliva."
+	cost = 0
+	var_changes = list(
+		"is_vampire" = TRUE,
+		"darksight" = 7,
+		"flash_mod" = 3.0,
```

Comment:
set this to 2 not 3.0

numbers are implicitly floats in byond
also 3 is insane